### ISSUE TITLE
aide

### ISSUE THREAD

**Issue by aurlipat** [N0rz3/Phunter#5]
Je ne comprends pas comment faire pour ouvrir le logiciel, je suis un peu novice ! 
Possible de me contacter en MP ?

**Comment by N0rz3** [comment]
Salut, alors toutes les commandes pour l'installation et le lancement sont dans le readme donc en gros quand tu l'as clone ou unzip dans ton repertoire tu le lances avec `python phunter.py -h` pour voir toutes les commandes

**Comment by N0rz3** [comment]
et après, tu le lances avec soit `python phunter -t +33etc...` ou `python phunter.py -f file.txt` (le fichier doit contenir les nums genre :
```
+33etc...
+33etc...
+33etc...
+33etc...
```

après ta plein d'autres arguments comme le `-p`, `-a`, etc...

**Comment by aurlipat** [comment]
Bonsoir,
Merci pour ce retour,
Je débute totalement.
Et je ne sais pas comment cloner. Et ce qu est un répertoire !
Si tu peux vite fait m expliquer. Je ne demande qu a apprendre

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by aurlipat** [comment]
Quand j'essai de l'ouvrir il disparait immediatement

**Comment by N0rz3** [comment]
Alors déjà, est-ce-que tu l'as ouvert avec la commande que je t'ai donnée si dessus (`python phunter.py -h`) ? Et as-tu installé les requirements avec la commande `pip install -r requirements.txt` ?

**Comment by aurlipat** [comment]
Oui, j'ai fait ce que tu m'as dit ... je pense que je ne me suis pas trompé

**Comment by N0rz3** [comment]
Ok, as-tu X pour passer mp ?

**Comment by aurlipat** [comment]
@aurel8456

**Comment by aurlipat** [comment]
???? Peux tu me donner des nouvelles merci

**Comment by N0rz3** [comment]
Salut, désolé mais en ce moment je suis assez occupé et je ne suis pas disponible pour le moment je te contacterai pour t'aider quand je pourrai
